### DISCUSSION TITLE
Ranging Configs

### DISCUSSION THREAD

**Discussion by mlayou** [android/nowinandroid#1071] [Q&A]
Hi, 

As far as I understand it, the current uwb library supports only one ranging configuration, namely CONFIG_UNICAST_DS_TWR() = 1 which implements a DS-TWR ranging
 with a **STATIC STS**. The library has other pre-defined Configs, but if I'm not mistaken they are not currently implemented. Correct?

Some of those future configs (for example, CONFIG ID 4, 5, and 7) refer to a **P-STS security mode**. I don't find this terminology in the IEEE UWB documentation or the FiRa documentation. Is this "**P-STS security mode**" just another name for "**Dynamic STS**"? If not, can you provide a link to where P-STS mode is defined? And can you confirm that "P-STS" stands for "Provisioned STS"?

Thank you.

**Comment by alexvanyo** [comment]
This doesn't seem to be related to the Now in Android project, if this is an issue for an androidx library please create a report on the Google issue tracker for the library.
